test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,107 @@
+
+    ```bash
+    aws ec2 describe-images --region <region_name> --filters "Name=architecture,Values=x86_64" "Name=name,Values=ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-*"
+    ```
+
+    Upon completion of the script's execution, an Ansible inventory file containing the IP addresses of the master and worker nodes will be generated in the present directory at `.data/hosts_core`.
+
+2. Deploy a Kubernetes cluster:
+
+    ```bash
+    ansible-playbook -i .data/hosts_core deploy-masters.yaml --ssh-common-args='-o StrictHostKeyChecking=no'
+    ansible-playbook -i .data/hosts_core deploy-workers.yaml --ssh-common-args='-o StrictHostKeyChecking=no'
+    ```
+
+3. Deploy KubeStellar in the hosting cluster:
+
+    ```bash
+    ansible-playbook -i .data/hosts_core deploy_ks_core.yaml --ssh-common-args='-o StrictHostKeyChecking=no' -e 'ks_release=0.26.0'
+    ```
+
+    You can use the variable `ks_release` to specify the KubeStellar release. Kubestellar is deployed using the [KS helmchart](https://github.com/kubestellar/kubestellar/tree/release-0.26.0/core-chart) configured with a ITS of type host. 
+
+4. Create the WEC hosting instances:
+
+    ```bash
+    ansible-playbook create-ec2.yaml -e "cluster_name=wec region=us-east-2 vpc_name=<vpc_name> aws_key_name=mykey wecs_hosting_instances=1 instance_type=t2.xlarge archt=x86_64 ec2_image='<your-aws-ami>" 
+    ```
+
+    Use the variable `wecs_hosting_instances` to specify the number of ec2 instances to be created to host WEC kind clusters.
+    
+    Upon completion of the script's execution, an Ansible inventory file containing the IP addresses of the ec2 WEC hosting instances will be generated in the present directory at `.data/hosts_wec`.
+
+5. Create the WEC kind clusters:
+
+    a) Update the inventory of the WEC instances to include the IP address of a master node from the K8s cluster created in step 1 above:
+
+    First, find the required master's IP address info in the following file: `.data/hosts_core`.
+
+    Next, use the following command to see the content of the WEC ansible inventory:
+    ```bash
+    cat .data/hosts_wec
+    ```
+
+    Sample output: 
+    ```console
+    [masters]
+    <add master node info here!>
+
+    [add_workers]
+    worker1 ansible_host=192.168.56.1
+    ```
+
+    Lastly, edit the contents of the file `.data/hosts_wec` to include the IP address of a master node:
+    ```bash
+    vi .data/hosts_wec
+    ```
+
+    b) Create Kind cluster WECs and connect to KS Core cluster
+
+    ```
+    ansible-playbook -i .data/hosts_wec deploy_ks_wec.yaml --ssh-common-args='-o StrictHostKeyChecking=no' -e 'num_wecs=1'
+    ```
+
+    Use the input paramater `num_wecs` to specify the number of kind clusters to be created for each WEC Hosting Instances. The above command creates kind WEC clusters and connects them to the KubeStellar core cluster created in step 1. Furthermore, it attaches a [KWOK](https://github.com/kubernetes-sigs/kwok) fake node to each kind cluster.
+
```

Comment:
I think this doc should be broken into two sections, one for setup and one for teardown.

Reply:
Fixed! This doc is now broken into the following sections: a) Deployment and b) Uninstall